packages/docusaurus-plugin-content-docs/src/theme/DocItem/index.js

```diff
@@ -32,32 +32,34 @@ function DocItem(props) {
   const {metadata, content: DocContent, docsMetadata} = props;
 
   return (
-    <div className={styles.docBody}>
+    <div>
       <Head>
         {metadata && metadata.title && <title>{metadata.title}</title>}
       </Head>
-      <div className={`${styles.container} container margin-bottom--lg`}>
+      <div className="padding-vert--lg">
         <div className="row">
-          <div className="col col--8">
-            <header>
-              <h1 className="margin-bottom--lg">{metadata.title}</h1>
-            </header>
-            <article>
-              <div className="markdown">
-                <DocContent />
+          <div className="col">
+            <div className={styles.docItemContainer}>
```

Comment:
Possible to combine this class name with `col` to save one layer of wrapping?

Reply:
Nope this is intentional. `col` itself has some `padding` styles and `docItemContainer` also has `padding` styles. Then the winner will be determined by the order in the CSS and since one is a CSS module the other is a root stylesheet, the order is not guaranteed.

Reply:
I see, thanks for the explanation!